engines/composer/detection_tables.h

```diff
@@ -148,12 +148,12 @@ static const ComposerGameDescription gameDescriptions[] = {
 			"imoking",
 			0,
 			{
-				{"book.mac", GAME_CONFIGFILE, "d09a628d147506be1dee854bc9f66664", 102},
+				{"book.mac", GAME_CONFIGFILE, "d:4896a22874bb660f5ba26a0af111f9c0", 1868},
```

Comment:
Is it data fork on resource fork?

Reply:
I change from resource to data fork as we only use data fork in the actual code